doc/source/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
```

Comment:
Just to open the discussion, do we want all these comments that are autogenerated? they may contain useful information, but without them the file would be much more compact, and things would probably be easier to find.

Reply:
I'd keep this particular comment for now as I find it helpful.